src/contracts/contract_deployer.star

```diff
@@ -9,10 +9,18 @@ FUND_SCRIPT_FILEPATH = "../../static_files/scripts"
 utils = import_module("../util.star")
 
 
+CANNED_VALUES = (
```

Comment:
We should really make these optional in the rollup config so that they are overrides with defaults rather than needing to redefine the same values everywhere

Reply:
agreed